users/app.py

```diff
@@ -40,17 +40,20 @@ def get_authorization():
 @app.route('/v1/', methods=['PUT'])
 def add_update_user():
     """Add or update the user in the db and returns new user object."""
-    user = request.getJSON()
-    if user is None:
-        # TODO(mukobi) validate the user object has everything it needs
-        return jsonify(error="You must supply a valid user in the body")
-    # TODO(mukobi) add or update the user in the database
-    added_new_user = True
-    # TODO(mukobi) get the new user object from the db to return
-    user_object = {
-        "user_id": "0", "username": "Dummy User", "edit_access": False
-    }
-    return jsonify(user_object), (201 if added_new_user else 200)
+    gauth_token = request.form.get('gauth_token')
+    if gauth_token is None:
+        return "Error: You must supply a valid gauth_token.", 400
```

Comment:
Is this a message that the end user sees? Something like "Error: You must authenticate through google" might be more actionable.

Reply:
Done https://github.com/knative-portability/large-events/pull/30/commits/c48a0d4f4b29ac484a58331f88fb88a0f459f651